includes/template-tags.php

```diff
@@ -330,7 +330,7 @@ function dokan_get_dashboard_nav() {
     );
 
     $settings = array(
-        'title' => __( 'Settings <i class="fa fa-angle-right pull-right"></i>', 'dokan-lite'),
+        'title' => sprintf(__("%s <i class='fa fa-angle-right pull-right'></i>"), __( 'Settings', 'dokan-lite')),
```

Comment:
Don't need to add `__()` function inside `sprintf` function.

Reply:
Thanks for the feedback, I corrected it. Please check.

Reply:
Use single quote outside and double quote for attributes. like 
`'%s <i class="fa fa-angle-right pull-right"></i>'`

Reply:
Please also remove extra `()` inside `sprintf`